bioimageio/core/_resource_tests.py

```diff
@@ -35,14 +36,21 @@ def test_model(
     source: Union[v0_5.ModelDescr, PermissiveFileSource],
     weight_format: Optional[WeightsFormat] = None,
     devices: Optional[List[str]] = None,
-    decimal: int = 4,
+    absolute_tolerance: float = 1e-4,
+    relative_tolerance: float = 1e-1,
```

Comment:
0.1 seems a bit too generous for a relative tolerance... especially considering we had 0 before and were mostly fine...
```suggestion
    relative_tolerance: float = 1e-4,
```

Reply:
I can make the change, these were mostly placeholder defaults before getting some opinions.

I was going off the [numpy defaults](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html) where the relative tolerance is `1e3` times less than the absolute tolerance. But I also chose it kinda arbitrarily.